packages/common/sagas/workflow.js

```diff
@@ -976,10 +978,11 @@ function* setWorkInProgressItemSaga({ payload: { item } }) {
 
 function* initDashboardSaga() {
   try {
+    yield put(vaultStartLoading());
     yield takeLatest(VAULTS_ARE_READY, openCurrentVaultSaga);
+    yield put(setWorkInProgressItem(null));
     yield call(initWorkflowSaga);
     yield call(checkTeamsKeyPairs);
-    yield call(checkWIPItem);
```

Comment:
Why did you remove this function call?

Reply:
We need to reset wip-item while changing vault instead of update it